### ISSUE TITLE
Dark Mode themes in Ubuntu 18.04 have no effect on Corebird UI

### ISSUE THREAD

**Issue by jerblack** [baedert/corebird#833]
I read from previous closed issues that Corebird is supposed to adapt to the dark theme when set by the Gnome Tweak Tool or by enabling a Dark theme. I Just installed Ubuntu 18.04 LTS and enabled the inbox Adwaita-Dark which made the main Files app dark as well as the Tweak Tool itself, but not Corebird. Is there another step I am missing?

**Comment by baedert** [comment]
Screenshot please? That should just work.

**Comment by jerblack** [comment]
screenshot:
https://i.imgur.com/UqzobLA.jpg

**Comment by Vistaus** [comment]
I've had that happen a couple of times on various distros. Setting the dark theme, logging out/back in or rebooting fixed the problem.

**Comment by jerblack** [comment]
New information and progress. I tried logging out and back in @Vistaus but it had no effect. However, after rebooting and logging back in I could no longer even open Corebird. I had originally installed from the snap package via the Ubuntu Software center, but I removed the snap and just installed via 'apt install corebird' and not only could Corebird launch at that point, I also got dark mode working in the apt version. It seems like my dark theme problem only occurs in the snap.
